src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -201,6 +201,20 @@ You can pass many different options to tracing, but it comes with reasonable def
 - If available: Browser Resources (Scripts, CSS, Images ...)
 - If available: Browser Performance API Marks
 
+*tracingOrigins Option*
+ 
+The default value of `tracingOrigins` is : `new ApmIntegrations.Tracing({tracingOrigins: ['localhost', /^\//]})`. This options will attach the `sentry-trace` header to all outgoing XHR/fetch requests that contain a string in the list or match the regex. In case your frontend is making requests to a different domain you might want to add it there so we attach the `sentry-trace`.
+
+*Example:*
+
+- Your frontend is running `mywebapp.com`
+- Your backend is running `api.mybackend.com`
+- You setup your frontend to make API calls to `api.mybackend.com`
+- Then the option need to be configured like this `new ApmIntegrations.Tracing({tracingOrigins: ['localhost', /^\//, 'api.mybackend.com']})`
```

Comment:
What's the `/^\//` for in this case?
Also, I'd not include `localhost`, because what's the point of sending the header to `localhost` (the page visitor's loopback device). Users may need to arrange for this config to have different values depending on environment.

Reply:
I would suggest adding a new bullet point before this one:

```
- In development, your API is at `localhost`
```

To explain what the `localhost` is needed for. Then they can make an informed choice on which values to include in their config

Reply:
The choice of hosts is very env specific. If you're developing on local containers, `localhost` is probably the wrong value. We can't know what the right value is for everyone, but we can explain what it needs to match and how to detect errors/ fix it.

Reply:
@rhcarvalho I think 
it's for checking that the string 'localhost' is validated or not by the regex /^\// ?

check out the function isMatchingPattern in string.ts
```
// tracing.ts
            const isWhitelisted = self._options.tracingOrigins.some((origin: string | RegExp) =>
              isMatchingPattern(url, origin),
            );
```

looks like you can also write ['localhost',  'string']?
```
// string.ts
export function isMatchingPattern(value: string, pattern: RegExp | string): boolean {
  if (isRegExp(pattern)) {
    return (pattern as RegExp).test(value);
  }
  if (typeof pattern === 'string') {
    return value.includes(pattern);
  }
  return false;
}
```

but i'm a little confused because:
```
reg = /^\// 
reg.test('foobar')
false
reg.test('foo/bar')
false
```

Reply:
@rhcarvalho nevermind, the regex is checking that the string doesn't begin with a forward slash.